src/anm/analytics.js

```diff
@@ -0,0 +1,76 @@
+var engine = require('engine');
+
+var Analytics = function () {
+    var self = this,
+        supportSendBeacon = !!navigator.sendBeacon,
+        timeout = supportSendBeacon ? 2000 : 1000,
```

Comment:
well, yeah. But it is supported by blink based browsers and firefox. And api compatibly are unlikely to broken, since it only sends POST request, what breaking change can be there?